app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+end
```

Comment:
- [ ] According to the project requirement, there should be a method that returns 3 most recent posts for a user. Kindly add that 👍🏽

👇🏾

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```